src/common/disk.rs

```diff
@@ -133,7 +134,9 @@ impl Disk {
         self.inner.is_read_only()
     }
 
-    /// Updates the disk' information.
+    /// Updates the disk' information with everything loaded.
+    ///
+    /// Equivalent to [`Disk::refresh_specifics`]`(`[`DiskRefreshKind::everything`]`())`.
```

Comment:
I tried this, and it doesn't render either the code or the link correctly :(

<img width="733" alt="Screenshot 2024-11-26 at 9 56 56 AM" src="https://github.com/user-attachments/assets/94b9afac-e852-4ea0-ae5e-f90e708b56ec">

Reply:
Oh, it's because of the paren.

Try this:

```suggestion
/// Equivalent to <code>[Disk::refresh_specifics](Disk::refresh_specifics)([DiskRefreshKind::everything](DiskRefreshKind::everything])())</code>.
```

?

Reply:
Oh, thanks for the pointer! Backslashing the paren works, which is even better :)

Reply:
Done!

Reply:
It's indeed much better. I'm ashamed for not thinking about it. 😅